packages/proteus/src/main/errors/DecryptError.ts

```diff
@@ -17,86 +17,84 @@
  *
  */
 
-import {ProteusError} from './ProteusError';
+import * as ProteusError from '../errors/ProteusError';
 
-export class DecryptError extends ProteusError {
-  static CODE = {
-    CASE_200: 200,
-    CASE_201: 201,
-    CASE_202: 202,
-    CASE_203: 203,
-    CASE_204: 204,
-    CASE_205: 205,
-    CASE_206: 206,
-    CASE_207: 207,
-    CASE_208: 208,
-    CASE_209: 209,
-    CASE_210: 210,
-    CASE_211: 211,
-    CASE_212: 212,
-    CASE_213: 213, // REMOTE_ENCRYPTION_FAILURE: Created by the receiver when getting a decryption failure symbol (💣)
-  };
+export enum CODE {
+  CASE_200 = 200,
+  CASE_201 = 201,
+  CASE_202 = 202,
+  CASE_203 = 203,
+  CASE_204 = 204,
+  CASE_205 = 205,
+  CASE_206 = 206,
+  CASE_207 = 207,
+  CASE_208 = 208,
+  CASE_209 = 209,
+  CASE_210 = 210,
+  CASE_211 = 211,
+  CASE_212 = 212,
+  CASE_213 = 213, // REMOTE_ENCRYPTION_FAILURE: Created by the receiver when getting a decryption failure symbol (💣)
```

Comment:
```suggestion
  /** REMOTE_ENCRYPTION_FAILURE: Created by the receiver when getting a decryption failure symbol (💣) */
  CASE_213 = 213,
```

Reply:
This will help VS Code rendering the info, right?

Reply:
Exactly :)